src/app/activityindicator/activityindicator.component.ts

```diff
@@ -11,7 +11,7 @@ import { Event, Router, NavigationEnd } from '@angular/router';
 export class ActivityindicatorComponent implements OnInit {
     // Sets the Position of Activity Indicator When Side bar is toggled.
     private static _localStorageKey = 'pcprepkitUser';
-    public position = 'col-md-3 col-md-offset-1 col-xs-3';
+    public position = 'col-md-3 col-md-OFFSET-1 col-xs-3';
```

Comment:
why suddenly caps ?

Reply:
@lunayach  This is the only thing I am waiting for in this PR.. once you reply to it, I will be able to approve or suggest changes..

Reply:
@Raxerz  you have any clue why this change has been made ?